### ISSUE TITLE
Missing vanilla level checks stuff / workarounds

### ISSUE THREAD

**Issue by gheskett** [HackerN64/HackerSM64#686]
- The endless staircase makes it impossible to get to BitS without a BLJ unless vanilla checks are enabled. This should be modified such that the endless staircase code is attached to the vanilla script instead so that it's removed by a level export rather than the define.
- `COURSE_BITDW` and `COURSE_BITFS` course success warp behavior is hardcoded in multiple locations when it instead could check for whether the last collected item was a key in each scenario.

**Comment by Mineqwerty** [comment]
Agreed, the only way I personally think this should and could be addressed in 2.1 is a first build console message as well as important notes in the readme
